melpa-packages.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env nix-shell
-#!nix-shell --pure -i bash -A env
+#!nix-shell -i bash -A env
 
 # usage: ./melpa-packages.sh --melpa PATH_TO_MELPA_CLONE
 
+## env var for curl
+export SSL_CERT_FILE=/etc/ssl/certs/ca-certificates.crt
```

Comment:
@bendlas Can we inherit `SSL_CERT_FILE` from the parent shell? Also, with this change, is it still necessary to set `SSL_CERT_FILE` in `Distribution.Nix.Fetch`?